src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchFarmConfig.java

```diff
@@ -0,0 +1,52 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.HasLegacyId;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneswitchFarmConfig {
+    @Expose
+    @ConfigOption(name = "Amount of Plots", desc = "The amount of plots the current Farm has.")
+    @ConfigEditorSlider(
+        minValue = 1,
+        maxValue = 5,
+        minStep = 1
+    )
+    public int plotAmount = 1;
+
+    @ConfigOption(name = "Farm Direction", desc = "The Direction you face while you are farming.")
```

Comment:
```suggestion
    @ConfigOption(name = "Farm direction", desc = "The Direction you face while you are farming.")
```

Reply:
"The direction you are facing while farming” sounds more natural to me.